tasks/main.yml

```diff
@@ -20,5 +20,20 @@
   tags:
     - haproxy
 
+- name: Make sure service dropin directory is available
+  file:
+    path: /etc/systemd/system/marathon.service.d/
+    state: directory
+  when: ansible_service_mgr == 'systemd'
```

Comment:
But what if it gets disabled in a later stage? The files are then not removed and (partial) SSL support is left behind.